clients/src/main/java/org/apache/kafka/clients/admin/internals/AdminMetadataManager.java

```diff
@@ -289,17 +304,24 @@ public void update(Cluster cluster, long now) {
 
         this.state = State.QUIESCENT;
         this.fatalException = null;
+        this.metadataAttemptStartMs = Optional.empty();
 
         if (!cluster.nodes().isEmpty()) {
             this.cluster = cluster;
         }
     }
 
+    public void initiateRebootstrap() {
+        requestUpdate();
+        this.metadataAttemptStartMs = Optional.of(0L);
```

Comment:
Question: Why don't we set the time to `now` as we do in NetworkClient?

Reply:
In rebootstrap(), we set time to `now` as in NetworkClient. In both, when we receive REBOOTSTRAP_REQUIRED error code, we set to 0, so that rebootstrap is triggered on the next poll regardless of time.